br/pkg/restore/client.go

```diff
@@ -806,6 +806,12 @@ func (rc *Client) RestoreRaw(
 		logutil.Key("startKey", startKey),
 		logutil.Key("endKey", endKey),
 	)
+
+	for _, f := range files {
+		summary.CollectSuccessUnit(summary.TotalKV, 1, f.TotalKvs)
+		summary.CollectSuccessUnit(summary.TotalBytes, 1, f.TotalBytes)
+	}
+
```

Comment:
Can you add some tests?

Reply:
yes, I should do it, but I have no ideas to construct tests.